environments/env_generator/examples/make_hotdog.json

```diff
@@ -0,0 +1,84 @@
+{
+    "version": "1.0.0",
+    "width": 3,
+    "height": 3,
+    "config": {
+        "cook_time": {
+            "hotdog": 3,
+            "bun": 3,
+            "default": 3
+        }
+    },
+    "stations": [
+        {
+            "name": "table",
+            "x": 0,
+            "y": 1
+        },
+        {
+            "name": "grill",
+            "x": 2,
+            "y": 1,
+            "id": "A"
+        },
+        {
+            "name": "table",
+            "x": 0,
+            "y": 0
+        },
+        {
+            "name": "grill",
+            "x": 2,
+            "y": 0,
+            "id": "B"
+        }
+    ],
+    "items": [
+        {
+            "name": "bun",
+            "x": 0,
+            "y": 1,
+            "stack-level": 0,
+            "predicates": ["istoastable"],
+            "id": "a"
+        },
+        {
+            "name": "hotdog",
+            "x": 0,
+            "y": 0,
+            "stack-level": 0,
+            "predicates": ["isgrillable"],
+            "id": "b"
+        }
+    ],
+    "players": [
+        {
+            "name": "robot",
+            "x": 1,
+            "y": 0,
+            "direction": [-1, 0]
+        }
+    ],
+    "goal": [
```

Comment:
add a goal description here as well. make sure to include that this is a toasted hotdog (i'd change the name of the environment to make this obvious `make_toasted_hotdog`)